strings/piglatinGame.java

```diff
@@ -0,0 +1,34 @@
+import java.util.Arrays;
+public class piglatinGame 
+{
+    public static void main(String[] args) 

```

Comment:
main method should not used. Create function and make it public so as to access.